mlir/lib/Conversion/FuncToLLVM/FuncToLLVM.cpp

```diff
@@ -544,6 +544,12 @@ struct CallOpInterfaceLowering : public ConvertOpToLLVMPattern<CallOpType> {
         callOp.getLoc(), packedResult ? TypeRange(packedResult) : TypeRange(),
         promoted, callOp->getAttrs());
 
+    newOp->setAttr(newOp.getOperandSegmentSizesAttrName(),
+                   rewriter.getDenseI32ArrayAttr(
+                       {static_cast<int32_t>(promoted.size()), 0}));
+    newOp->setAttr(newOp.getOpBundleSizesAttrName(),
+                   rewriter.getDenseI32ArrayAttr({}));
```

Comment:
This should not necessary if the `OptionalProperty` is deleted.
(and would otherwise better be spelled as `newOp.properties.op_bundles_sizes = {};` I believe)

Reply:
Updated.